concepticondata/conceptlists/DellaRosa-2010-417.tsv

```diff
@@ -0,0 +1,418 @@
+DellaRosa-2010-417-268	268	Cuore	heart	1223	HEART	625	668	623	230	626	212	356
+DellaRosa-2010-417-269	269	Denaro	money	1559	MONEY	624	668	621	276	631	145	247
+DellaRosa-2010-417-270	270	Diamante	diamond	3878	DIAMOND	650	637	505	340	612	126	356
+DellaRosa-2010-417-271	271	Divano	couch	3099	SOFA	694	676	679	174	700	100	153
+DellaRosa-2010-417-272	272	Elefante	elephant	1290	ELEPHANT	669	695	482	210	618	100	296
+DellaRosa-2010-417-273	273	Elicottero	helicopter	2580	HELICOPTER	675	689	468	220	585	106	268
+DellaRosa-2010-417-274	274	Equipaggio	crew			519	521	350	365	521	215	500
+DellaRosa-2010-417-275	275	Erba	grass	606	GRASS	695	691	663	167	656	113	313
+DellaRosa-2010-417-276	276	Esercito	army	1561	ARMY	621	618	507	325	608	157	394
+DellaRosa-2010-417-277	277	Falce	sickle	341	SICKLE	662	632	387	365	575	129	288
+DellaRosa-2010-417-278	278	Famiglia	family	194	FAMILY	541	629	682	192	680	212	206
+DellaRosa-2010-417-279	279	Ferita	wound	1109	WOUND	625	642	500	230	618	171	168
+DellaRosa-2010-417-280	280	Ferro	iron	621	IRON	661	633	573	278	587	154	175
+DellaRosa-2010-417-281	281	Fiamma	flame	889	FLAME	652	687	541	266	652	175	135
+DellaRosa-2010-417-282	282	Flotta	fleet			600	644	442	330	582	163	375
+DellaRosa-2010-417-283	283	Folla	crowd	2023	CROWD	581	674	558	319	647	221	297
+DellaRosa-2010-417-284	284	Fontana	fountain	2144	SPRING OR WELL	685	676	561	209	613	105	147
+DellaRosa-2010-417-285	285	Forbici	scissors	356	SHEARS	697	700	644	222	661	146	130
+DellaRosa-2010-417-286	286	Forchetta	fork	1302	FORK	695	697	689	158	674	106	313
+DellaRosa-2010-417-287	287	Formaggio	cheese	1569	CHEESE	677	700	637	150	682	112	144
+DellaRosa-2010-417-288	288	Foschia	mist	249	FOG	556	548	436	406	506	220	342
+DellaRosa-2010-417-289	289	Fragola	strawberry	3138	STRAWBERRY	700	700	659	170	632	100	136
+DellaRosa-2010-417-290	290	Fratello	brother	1262	BROTHER	641	641	647	200	600	124	159
+DellaRosa-2010-417-291	291	Fumo	smoke	778	SMOKE (EXHAUST)	613	647	586	245	618	182	252
+DellaRosa-2010-417-292	292	Fungo	mushroom	641	MUSHROOM	700	658	527	205	606	109	208
+DellaRosa-2010-417-293	293	Gallina	chicken	1318	CHICKEN	687	700	591	178	661	125	140
+DellaRosa-2010-417-294	294	Gallo	cock	1511	ROOSTER	700	695	486	215	600	112	212
+DellaRosa-2010-417-295	295	Ghiaccio	ice	617	ICE	679	679	580	225	608	119	171
+DellaRosa-2010-417-296	296	Giacca	jacket	2770	JACKET	695	674	653	208	670	100	206
+DellaRosa-2010-417-297	297	Giardino	garden	586	GARDEN	695	656	665	175	645	121	238
+DellaRosa-2010-417-298	298	Giraffa	giraffe	3089	GIRAFFE	681	674	463	181	644	106	306
+DellaRosa-2010-417-299	299	Guerriero	warrior	3608	WARRIOR	563	616	395	270	518	203	436
+DellaRosa-2010-417-300	300	Gufo	owl	735	OWL	685	676	480	265	625	124	244
+DellaRosa-2010-417-301	301	Infermiera	nurse	1104	NURSE	630	640	579	285	631	125	317
+DellaRosa-2010-417-302	302	Ingegnere	engineer			621	544	530	456	556	137	400
+DellaRosa-2010-417-303	303	Insalata	salad			685	687	684	200	676	109	147
```

Comment:
Perhaps a good candidate for a new concept. Already a number of entries in previous concept lists.

Reply:
@MiraAhmedovic Could you open an issue linking this PR where we can decide whether we want to include SALAD as a new concept?